source/blog/2017-12-04-lvm-configuration-the-easy-way.html.md

```diff
@@ -0,0 +1,107 @@
+---
+title: LVM Configuration the Easy Way
+author: jmarks, nsofer
+tags: LVM, VDSM
+date: 2017-12-04 09:00:00 CET
+---
+
+Now oVirt features a simple way to prevent a rebooted host from scanning and then activating logical volumes that are not required directly by the host. Why? Because scanning and activating other logical volumes may cause data corruption, slow boot, and other issues.
```

Comment:
Now oVirt features a simple way to prevent a host from scanning and then activating logical volumes that are not required directly by the host. In particular, logical volumes on shared storage manged by oVirt, and logical volumes created by a guest in oVirt raw volumes. Why? Because scanning and activating other logical volumes may cause data corruption, slow boot, and other issues.